src/pages/Portfolio/ImpactDashboard/YourTeams.vue

```diff
@@ -69,7 +69,7 @@
 		<!-- all your teams link -->
 		<p v-if="totalTeams > 5" class="tw-text-center tw-mt-4">
 			<router-link
-				to="/blog/myloans"
```

Comment:
This link does not exist

Reply:
It should. are you saying it doesn't work?

Reply:
The page exists if you're logged in. Prob just needs to be an href.

Reply:
@mcstover Oh yeah I see, the link does exist in production. It seems to not resolve in dev: http://development.kiva.org/blog/myloans or in my tilt vm, but probably because of a missing entry in the tilt htaccess